code/game/objects/effects/temporary_visuals/miscellaneous.dm

```diff
@@ -141,6 +141,23 @@
 	. = ..()
 	update_icon()
 
+/obj/effect/temp_visual/bsa_splash
+	name = "Bluespace energy wave"
```

Comment:
I believe our style is that "Bluespace" is not a proper noun, so it should just be written "bluespace" here.

You can also use the [\improper marker](http://www.byond.com/docs/ref/info.html#/DM/text/macros) if you want something to remain capitalized but be rendered as an improper noun (which this is), e.g. `"\improper Bluespace energy wave"`.